src/ovirtapi/transform.js

```diff
@@ -486,6 +486,7 @@ const Snapshot = {
   toApi ({ snapshot }: { snapshot: SnapshotType }): ApiSnapshotType {
     return {
       description: snapshot.description,
+      persist_memorystate: snapshot.persistMemoryState,
```

Comment:
```
Cannot return object literal because boolean [1] is incompatible with string [2] in property persist_memorystate.

     src/ovirtapi/transform.js
     486|   toApi ({ snapshot }: { snapshot: SnapshotType }): ApiSnapshotType {
     487|     return {
     488|       description: snapshot.description,
     489|       persist_memorystate: snapshot.persistMemoryState,
     490|     }
     491|   },
     492| }

     src/ovirtapi/types.js
 [2]  50|   persist_memorystate?: string
        :
 [1]  59|   persistMemoryState: boolean,
```

Reply:
This error makes sense to me, especially when I look at https://github.com/oVirt/ovirt-web-ui/blob/master/src/ovirtapi/transform.js#L481

Reply:
Great input :)
Thanks.